packages/calling-component-bindings/review/beta/calling-component-bindings.api.md

```diff
@@ -80,9 +82,12 @@ export type CallingHandlers = {
     onStopScreenShare: () => Promise<void>;
     onToggleScreenShare: () => Promise<void>;
     onHangUp: () => Promise<void>;
+    onHold: () => Promise<void>;
```

Comment:
`onToggleHold` might be a better name since the same method will toggle between the hold states.